src/app/api/login/route.ts

```diff
@@ -57,7 +57,7 @@ export const POST = withAppRouterHighlight(async (request: NextRequest) => {
     );
 
     if (!validPassword) {
-      return new Response("Wrong password or user does not exist!", {
+      return new Response("Wrong password!", {
```

Comment:
Same as above

Reply:
Make sure the error code is also the same, such that people cannot notice the difference